examples/python/grizzly/scripts/replicate-csv

```diff
@@ -4,6 +4,32 @@ import argparse
 import csv
 import random
 
+def transformCSV_birth_analysis(inputFile, outputFile, replicationFactor):
+    years = [str(i) for i in range(0, 1000 * replicationFactor)]
+    gender = [str(i) for i in range(0, 4)]
+    with open(inputFile, 'r') as f1, open(outputFile, 'w') as f2:
+        lineId = 0
+        numCols = 0
+        count = 0
+        for line in csv.reader(f1, delimiter=','):
+            if lineId == 0:
+                numCols = len(line)
+            if len(line) != numCols:
+                continue
+            line_ele_list = [str(line_ele) for line_ele in line]
+
+            for factor in range(0, replicationFactor):
+                count += 1
+                new_year = years[count % len(years)]
+                # This is a hack. We need to get to the bottom of pandas ordering on sorts
+                # which is causing grizzly to include "Leslyn" when pandas does not
+                # for birth analysis (sort on groupmerger).
+                num_diff = str(int(line_ele_list[2]) + count)
```

Comment:
I don't fully understand the reason for this hack

Reply:
When we do the groupby_sort, we are sorting by number of births. However, some names have the same number of births associated with it. We can't seem to match whatever ordering pandas is using when it places items with the same sort_key value. This causes taking topk in a sorted vector to be off.

Reply:
I think we should get rid of this snippet and just use the transformCSV function that's already there. We should figure out what's going on with the sort instead of changing the actual values of the dataset.